packages/plugin-core/src/_extension.ts

```diff
@@ -394,9 +394,22 @@ export async function _activate(
       previousGlobalVersion,
       currentVersion,
     });
+
+    // check if this is an install event, but a repeated one on a new instance.
+    let isNewInstanceInstall = false;
+
     // set initial install ^194e5bw7so9g
     if (extensionInstallStatus === InstallStatus.INITIAL_INSTALL) {
-      MetadataService.instance().setInitialInstall();
+      // even if it's an initial install for this instance of vscode, it may not be for this machine.
+      // in that case, we should skip setting the initial install time since it's already set.
+      const metadata = MetadataService.instance().getMeta();
+      if (metadata.firstInstall === undefined) {
```

Comment:
Any chance that during our transition, that someone who is an active user of Dendron doesn't have `metadata.firstInstall` defined yet? 

My preference would also to be check for the presence of the `dendron.uuid` file - if it exists, then also don't set to initial install.  That way, since the uniqueness of `track` is based on .uuid, we should ensure we're 100% aligned and we won't have old users showing up in the onboarding funnel.

Reply:
> Any chance that during our transition, that someone who is an active user of Dendron doesn't have metadata.firstInstall defined yet?

That's possible. That's why we have a temporary backfill logic that follows right after.
Though if an active user doesn't have it defined yet, and never went through that backfill, and also has installed a fresh instance of vscode, this would fill it up with the current time.

Your suggestion to check for the uuid would solve the problem though. So I'll go with that.